CHANGELOG.md

```diff
@@ -2,15 +2,10 @@
 
 Canonical reference for changes, improvements, and bugfixes for Boundary.
 
-## Next
-
 ### New and Improved
 
-* config: Add support for go-sockaddr templates to Worker and Controller
```

Comment:
Seems you wiped the previous changelog entries and the `Next` title

Reply:
Oops.  Ty!